src/_P036_FrameOLED.ino

```diff
@@ -607,6 +609,35 @@ boolean Plugin_036(uint8_t function, struct EventStruct *event, String& string)
 
     case PLUGIN_WRITE:
     {
+      String arguments = String(string);
+      String name;
+      // this was to manage multiple instances of the plug-in.
+      // You can also call it this way:
+      // [TaskName].OLEDFRAMEDCMD, 1,"Temp. is 19.9"
```

Comment:
Does that work at all?
The `[taskname]` part should be converted already when calling the write

Reply:
By the way, I don't think this filtering should be done in the plugin, but should be done in calling the write function.  (as also discussed in your other PR)

Reply:
> Does that work at all?
> The `[taskname]` part should be converted already when calling the write

I tested by connecting 2 displays (hence the remark about the soldering) to a wemos d1 clone, and I can now address each display separately.
Yes, it would be nice if this was handled by the ESPEasy core, but currently it isn't, AFAICS.
I'll investigate further, and try to move it where it should be (or fix if it's broken).